src/utils/daos.c

```diff
@@ -483,6 +483,7 @@ common_op_parse_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 		{"user",	required_argument,	NULL,	'u'},
 		{"group",	required_argument,	NULL,	'g'},
 		{"principal",	required_argument,	NULL,	'P'},
+		{"name",	required_argument,	NULL,	'n'},
```

Comment:
Some option name reflecting the fact it is a property could be better, like for attr ?

Reply:
Could change it to --prop if you think that's better.

Reply:
I would prefer personally, but may be wait for others reviewers feedback ...

Reply:
Can --properties= option be reused, including related code such as daos_parse_properties() and daos_parse_property() functions?

Reply:
I could go with that, my main note of caution is that "properties" is plural, implying that one could pass multiple properties. I don't think that makes much sense in the set-prop case.

Reply:
@kccain @bfaccini What do you think of changing the existing properties option to "--prop" instead? More general so it works better for both cases

Reply:
Actually, looking at the code, maybe a good idea to just re-use as-is. I'll change it.